packages/core/src/stories/announcements/announce-tegel.stories.ts

```diff
@@ -54,7 +54,7 @@ export const Tegel = {
                         <h4 class="tds-u-mb2">@scania/tegel 🧱</h4>
                         <p>
                             The new
-                            package includes all components as web components, removes the old "native" components and makes the
+                            package includes all components as web-components, removes the old CSS class components and makes the
```

Comment:
Why `-`? We are mixing a lot. Most places we use "web componenents". Should we change all to -?

Reply:
good point

Reply:
I think technically it should be Web Components (capical case) since it's a name. But I'll keep it lowercase for consistency.